wordle-web/components/ValidWord.vue

```diff
@@ -0,0 +1,35 @@
+<!--create an object that shows a list of valid words from the avalible guesses using ValidWordList-->
+<template>
+    <v-menu>
+        <template v-slot:activator="{ props }">
+            <v-btn v-bind="props" >
+                available words: {{ validWords.length }}
+            </v-btn>
```

Comment:
```suggestion
      <v-btn color="primary" v-bind="props"> available words: {{ validWords.length }} </v-btn>
```
Adding this will make it user more aware that this is clickable. In dark mode, the elevation of the button isn't as prominent.

Reply:
Agreed, thanks for the sugguestion.